Cargo.toml

```diff
@@ -90,17 +90,31 @@ is_mmtk_object = ["global_alloc_bit"]
 # Enable object pinning, in particular, enable pinning/unpinning, and its metadata
 object_pinning = []
 
-# The following two features are useful for using Immix for VMs that do not support moving GC.
+# The following features are useful for using Immix for VMs that do not support moving GC.
 
 # Disable defragmentation for ImmixSpace.  This makes Immix a non-moving plan.
 immix_no_defrag = []
 # Disable nursery copy for ImmixSpace. This along with immix_no_defrag makes StickyImmix a non-moving plan.
 immix_no_nursery_copy = []
 # Reduce block size for ImmixSpace.  This mitigates fragmentation when defrag is disabled.
 immix_smaller_block = []
+
+# The following features are useful for debugging moving GC when using Immix.
+# If the VM cannot use Semispace for this purpose (for example, if the VM needs object pinning),
+# these features will make Immix copy as many objects as possible.
+
+# Make every GC a defragment GC.
+immix_stress_defrag = []
```

Comment:
Why do you think we should have features rather than just having two `const boolean`s? I have no objection for the PR. But we should have a rule about what becomes a 'feature', and check through our feature list.

Reply:
Good point.  After a second thought, I think we don't need to make them features.

"non-moving Immix" is a feature because some VMs cannot support copying GC (in production or at some stage of development).  They need to be built against `mmtk-core` as is.  So we have `immix_no_defrag` so that they can disable defrag in the `Cargo.toml` of the VM binding.

Stress testing defragmentation should not be a feature because it is only used in testing environments, and never used in production.  It is always possible to clone `mmtk-core` to a local directory and modify the source code of `mmtk-core` when testing.  It is convenient to add a parameter that can be tuned in the VM binding repository without changing `mmtk-core`, but it is unnecessary.